### PR TITLE
Remove no proxy duplicates

### PR DESCRIPTION
**Pull request by riuvshin**
### What does this PR do?
remove duplicate entries in no_proxy because it may cause issues

#### Changelog
remove duplicate entries in no_proxy because it may cause issues

### CODE REVIEW

**Comment by benoitf** [comment]
Needs to update typo in changelog on "duplicate"

File: dockerfiles/init/modules/codenvy/templates/codenvy.env.erb [hunk 1]
```diff
@@ -50,7 +50,7 @@ JAVA_HTTPS_PROXY_PORT=-Dhttps.proxyPort=<%= @https_proxy_for_codenvy.gsub(/^http
 <% end -%>
 <% end -%>
 <% if ! @no_proxy_for_codenvy.empty? -%>
-JAVA_NO_PROXY=-Dhttp.nonProxyHosts='<%= @no_proxy_for_codenvy.gsub(/^https?\:\/\//, '').gsub(/^www./,'').gsub(',','|') %>|'
```
Comment (garagatyi) [inline dockerfiles/init/modules/codenvy/templates/codenvy.env.erb]: Looks like this statement can be simplified: 
`.split(",").uniq.join("|") %>|'`

File: dockerfiles/init/modules/codenvy/templates/machine.properties.erb [hunk 2]
```diff
@@ -86,7 +86,7 @@ che.docker.network_driver=<%= scope.lookupvar('codenvy::che_machine_docker_netwo
...
...it(':')[1].gsub(/\/.*/,'') %><% end -%><% end -%><% if ! @https_proxy_for_codenvy_workspaces.empty? -%><% if @https_proxy_for_codenvy_workspaces.include? '@' -%> -Dhttps.proxyUser=<%= @https_proxy_for_codenvy_workspaces.gsub(/^https?\:\/\//, '').gsub(/^www./,'').split('@')[0].split(':')[0] %> -Dhttps.proxyPassword=<%= @https_proxy_for_codenvy_workspaces.gsub(/^https?\:\/\//, '').gsub(/^www./,'').split('@')[0].split(':')[1] %> -Dhttps.proxyHost=<%= @https_proxy_for_codenvy_workspaces.gsub(/^https?\:\/\//, '').gsub(/^www./,'').split('@')[1].split(':')[0] %> -Dhttps.proxyPort=<%= @https_proxy_for_codenvy_workspaces.gsub(/^https?\:\/\//, '').gsub(/^www./,'').split('@')[1].split(':')[1].gsub(/\/.*/,'') %><% else -%> -Dhttps.proxyHost=<%= @https_proxy_for_codenvy_workspaces.gsub(/^https?\:\/\//, '').gsub(/^www./,'').split(':')[0] %> -Dhttps.proxyPort=<%= @https_proxy_for_codenvy_workspaces.gsub(/^https?\:\/\//, '').gsub(/^www./,'').split(':')[1].gsub(/\/.*/,'') %><% end -%><% end -%><% if ! @no_proxy_for_codenvy_workspaces.empty? -%> -Dhttp.nonProxyHosts='<%= @no_proxy_for_codenvy_workspaces.gsub(/^https?\:\/\//, '').gsub(/^www./,'').gsub(',','|').split("|").uniq.join("|") %>|'<% end -%>
```
Comment (garagatyi) [inline dockerfiles/init/modules/codenvy/templates/machine.properties.erb]: Looks like this statement can be simplified: 
`.split(",").uniq.join("|") %>|'<% end -%>`

**Comment by riuvshin** [comment]
@garagatyi updated

**Pull request merged**